examples/aggregator/module/rust/src/lib.rs

```diff
@@ -35,159 +35,142 @@ pub mod proto {
+        // The router node has a public confidentiality label, and therefore cannot read the
+        // contents of the request of the invocation (unless it happens to be public as well), but
+        // it can always inspect its label.
+        match (&command.receiver, &command.sender) {
+            (Some(receiver), Some(sender)) => {
+                let label = receiver.label()?;
+                let grpc_response_writer = oak::grpc::ChannelResponseWriter::new(sender.clone());
+                if !is_valid_label(&label) {
+                    grpc_response_writer
+                        .close(Err(oak::grpc::build_status(
+                            rpc::Code::InvalidArgument,
+                            "Invalid request label",
+                        )))
+                        .context("Couldn't send error response back")?;
+                    anyhow::bail!("Invalid request label: {:?}", label);
                 }
-            },
-            None => {
-                let err = "No data specified";
-                Err(grpc::build_status(grpc::Code::InvalidArgument, &err))
+
+                // Route gRPC invocations to Aggregator Node.
+                self.aggregator_sender
+                    .send(&Either::Right(command))
+                    .context("Couldn't send invocation to worker node")
+            }
+            _ => {
+                anyhow::bail!("Received malformed gRPC invocation");
             }
         }
     }
 }
 
-oak::entrypoint!(aggregator<Either<AggregatorInit, oak::grpc::Invocation>> => |receiver: Receiver<Either<AggregatorInit, oak::grpc::Invocation>>| {
-    oak::logger::init_default();
-
-    let mut receiver_iterator = receiver.iter();
-
-    // Receive the initialization message.
-    let init_message: AggregatorInit = receiver_iterator
-        .find_map(Either::left)
-        .expect("Couldn't receive init message");
-    let grpc_client_invocation_sender = init_message
-        .grpc_client_invocation_sender
-        .expect("Couldn't receive gRPC invocation sender")
-        .sender
-        .expect("Empty gRPC invocation sender");
-
-    // Run event loop and handle incoming invocations.
-    let node = AggregatorNode::new(oak::grpc::client::Client {
-        invocation_sender: grpc_client_invocation_sender,
-    });
-    let dispatcher = AggregatorDispatcher::new(node);
-    oak::run_command_loop(dispatcher, receiver_iterator.filter_map(Either::right));
-});
-
 #[derive(Debug, serde::Deserialize)]
 #[serde(deny_unknown_fields)]
 pub struct Config {
     pub grpc_server_listen_address: String,
-}
-
-/// Create initialization message for Aggregator Node.
-fn create_init_message(sender: &oak::io::Sender<GrpcInvocation>) -> AggregatorInit {
-    AggregatorInit {
-        grpc_client_invocation_sender: Some(GrpcInvocationSender {
-            sender: Some(sender.clone()),
-        }),
-    }
+    pub grpc_client_connect_address: String,
```

Comment:
I think this is the backend server address, right? If so, could you name it that way?